templates/shrine/src/shrine-detail.html

```diff
@@ -243,7 +254,8 @@
           <h1>[[item.title]]</h1>
           <p>[[item.description]]</p>
           <div class="quantitySelector">
-            <select>
+            <label class="sr-only" for="quantity">Number of items:</label>
+            <select id="quantity">
```

Comment:
@blasten:

1. with the same accessibility, I prefer the native element against the use of WAI-ARIA. 
2. yes, you save a node. But with this idea (and I think is _dangerous_) you can build a data table with `div` elements and use `aria` attributes (and believe me, I've seen components like this in github) to set the relations instead of use the native elements of a table, so you save more nodes in the DOM that using `th`, `td`, `tr`, etc...
3. I should have made the PR with the lable visible.